### DISCUSSION TITLE
How to disable 'apikey ' value ?

### DISCUSSION THREAD

**Discussion by arkadiusz-mazur-goespedi** [pimcore/pimcore#15533] [Q&A / Support]
In PimCore, in the "DataHub/Security definition" tab, there is an option:
'Skip Permission Check'.
I think it is responsible for disabling user verification by the 'apikey' value in GraphQL panel.
Unfortunately, after restarting the project, this option always has the default value,
it means it is turned on.
Do you know how permanently disable this option?

**Comment by dvesh3** [comment]
Hi @arkadiusz-mazur-goespedi the `Skip Permission Check` is used for skipping the `Workspaces` permissions. The datahub endpoint configurations can be stored differently based on the deployment needs. please see https://pimcore.com/docs/platform/Datahub/Deployment/ more details.

**Reply by arkadiusz-mazur-goespedi** [reply]
Thanks!
